Code/Engine/RendererCore/RenderWorld/Implementation/RenderWorld.cpp

```diff
@@ -580,10 +580,12 @@ void ezRenderWorld::Render(ezRenderContext* pRenderContext)
   }
 
   filteredRenderPipelines.Clear();
-
-  renderEvent.m_Type = ezRenderWorldRenderEvent::Type::EndRender;
-  EZ_PROFILE_SCOPE("EndRender");
-  s_RenderEvent.Broadcast(renderEvent);
+  /// NOTE: (Only Applies When Tracy is Enabled.)Tracy Seems to declare Timers in the same scope, so dual profile macros can throw: '__tracy_scoped_zone' : redefinition; multitple initalization, so we must scope the two events.
```

Comment:
Im unsure if it should scope here, but Tracy declares Timers in the same scope.

Reply:
Just looked a bit through the documentation and found this macro: ZoneNamedN(variableName, "Zone name"). This would allow us to use a custom variable name to prevent these scoping issues, like e.g. concat a fixed name with the line number like we do with the ez profiling and also pass it a custom name.

Reply:
Unresolving to add conversions for ez string types